index.html

```diff
@@ -3,9 +3,138 @@
   <head>
     <meta charset="utf-8">
     <title></title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="./assets/css/reset.css">
+    <link rel="stylesheet" href="./assets/css/style.css">
   </head>
   <body>
-    <script src="js/script.js" charset="utf-8"></script>
+
+  <div class="container">
+  <!-- Non box elements -->
+  <h1>Snakes and ladders</h1>
+
+  <!-- grid elements -->
+  <div class="wrapper">
+
+    <div class="box" data-id="100"></div>
```

Comment:
Recommend using a js loop to append these div elements to the .wrapper div. Data id can be added using jquery's .attr(), or .data(), or an id or class can also be used.

Reply:
append order can be 100-> 91, 81-> 90, 80-> 71 etc.